app/src/main/AndroidManifest.xml

```diff
@@ -23,9 +23,11 @@
             android:theme="@style/FullscreenTheme">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN"/>
-
+                <action android:name="android.hardware.usb.action.USB_DEVICE_ATTACHED" />
```

Comment:
you might missed 'feature' of USB hosting.
<uses-feature android:name="android.hardware.usb.host" />

see the document below
https://developer.android.com/guide/topics/connectivity/usb/host.html#manifest

Reply:
👍